content/en/blog/2024/scaling-opentelemetry-collectors.md

```diff
@@ -0,0 +1,152 @@
+---
+title: 
+  Manage OpenTelemetry Collectors at Scale with Ansible
+linkTitle: OTel Collector with Ansible
+date: 2024-03-12
+author: '[Ishan Jain](https://github.com/ishanjainn) (Grafana)'
```

Comment:
add a cSpell:ignore: section for words unknown to cspell, check other blog posts for examples